pkg/driverbuilder/builder/local.go

```diff
@@ -0,0 +1,53 @@
+package builder
+
+import (
+	_ "embed"
+	"fmt"
+	"github.com/falcosecurity/driverkit/pkg/kernelrelease"
+)
+
+// NOTE: since this is only used by local build,
+// it is not exposed in `target` array,
+// so no init() function to register it is present.
+
+//go:embed templates/local.sh
+var localTemplate string
+
+type LocalBuilder struct {
+	GccPath string
```

Comment:
`GccPath` is the full path to a discovered Gcc, and will be set by `local` build processor.

Reply:
Since there might be multiple `gcc` present, we tried all of them until we can build the module.